src/main/java/gov/cabinetoffice/gap/adminbackend/services/SpotlightSubmissionService.java

```diff
@@ -20,4 +25,30 @@ public SpotlightSubmission getSpotlightSubmission(UUID spotlightSubmissionId) {
                 "A spotlight submission with id " + spotlightSubmissionId + " could not be found"));
     }
 
+    public void addSpotlightBatchToSpotlightSubmission(UUID spotlightSubmissionId, SpotlightBatch spotlightBatch) {
```

Comment:
This method appears to be un-used. Is this for another PR?

Reply:
Yes, this is work from another ticket